### DISCUSSION TITLE
How to change NavigationBar color in landscape mode [Avalonia.Android]?

### DISCUSSION THREAD

**Discussion by J0nathan550** [AvaloniaUI/Avalonia#17001] [Q&A]
Hello people, I have been struggling with changing color of navigation bar. I have done a little bit of researcing on how to change the navigation bar and status bar color in Xamarin and managed to change it only for the vertical orientation like this:

```cs
            Window?.SetNavigationBarColor(Color.Black);
            Window?.SetStatusBarColor(Color.Black);
```

It looks and works like this:

![Screenshot_2](https://github.com/user-attachments/assets/d021deb0-98b8-4572-b4af-52a5c39e6460)

But if I switch to landscape this happens:

![Screenshot_3](https://github.com/user-attachments/assets/926c8721-5510-4076-be40-999af8966a3c)

Help me change this color in landscape mode please 😄 !

**Comment by emmauss** [comment]
Please use our InsetsManager if you want to change system decorations and behavior on Android.

**Reply by J0nathan550** [reply]
Can you send link? Or show an example?

**Comment by J0nathan550** [comment]
Figure out, here is what I did

`MainView.xaml`:
```xml
<UserControl xmlns="https://github.com/avaloniaui"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
             xmlns:vm="using:SoundScapes.ViewModels"
             xmlns:views="using:SoundScapes.Views"
			 Loaded="UserControl_Loaded"
			 mc:Ignorable="d" d:DesignWidth="400" d:DesignHeight="700"
             x:Class="SoundScapes.Views.MainView">
             ....
```

`MainView.xaml.cs`:
```cs
    private void UserControl_Loaded(object? sender, RoutedEventArgs e)
    {
        if (OperatingSystem.IsAndroid())
        {
            var insetsManager = TopLevel.GetTopLevel(this).InsetsManager;
            insetsManager.SystemBarColor = Colors.Black;
            insetsManager.DisplayEdgeToEdge = true;
        }
    }
```
Works perfectly fine!
